client/my-sites/stats/stats-tabs/index.jsx

```diff
@@ -27,13 +28,7 @@ export default React.createClass( {
 		let statsTabs;
 
 		if ( dataList ) {
-			let data = dataList.response.data.filter( function( item ) {
-				return item[ activeKey ] === activeIndex;
-			} );
-
-			if ( data.length ) {
-				data = data.pop();
-			}
+			const data = find( dataList.response.data, { [ activeKey ]: activeIndex } );
```

Comment:
>again might need `findLast`. but wow this change is glorious.

@timmyc Do you recall if we should ever expect there to be more than one match from this filter?